company_website/templates/common_partials/footer.haml

```diff
@@ -82,8 +82,16 @@
             .row
                 .col-lg-9.col-12
                     %span.copyright
-                        © 2019 Code Poets sp. z o.o. All rights reserved.  | NIP: 8952068133 | KRS: 0000598357 | REGON: 363584646
+                        ©
+                        %span{:id => "current_year_id"}
+                        %span
+                            Code Poets sp. z o.o. All rights reserved.  | NIP: 8952068133 | KRS: 0000598357 | REGON: 363584646
                 .col-lg-3.col-12
                     %nav.sub-menu-socket
                         %a.light-grey-link{:href => "{% url 'privacy_and_policy' %}"}
                             Privacy and Cookie Policy
+
+:javascript
+    var current_year = new Date().getUTCFullYear().toString()
+
+    document.getElementById("current_year_id").innerHTML =  current_year
```

Comment:
It must be in .js file. We should never use `javascript` tag in html itself - of course there are few cases when it is necessary but mostly we should use it. Please move it to .js file.

Reply:
Done